Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <RepositoryRoot>$(MSBuildThisFileDirectory)</RepositoryRoot>
```

Comment:
this is not used anywhere, right?

Reply:
I thought it was used as part of source link, but it appears that it can generate it itself these days, so I will remove :)